R/impact_central.R

```diff
@@ -142,9 +142,17 @@ impact_by_year_of_vaccination <- function(meta1, raw_impact, fvps, fvps_updates
     stop("method2a can not accommodate both routine and campaign impact in the same time, as routine and campaign impact are calculated differently.")
   }
   # prepare data
-  fvps$time <- fvps$year - fvps$age
-  fvps <- merge_by_common_cols(fvps, vaccine_delivery, all.y = TRUE)
   fvps <- fvps[fvps$year %in% vaccination_years, ]
+  fvps$time <- fvps$year - fvps$age
```

Comment:
should this method be exported? Is there a doc somewhere which connects between the mysterious 1/2a/2b notation and the names used here? (given that this throws with a "method 2a")

Reply:
One for @xiangli313 would be good to add any additional notes about methods to https://mrc-ide.myjetbrains.com/youtrack/articles/VIMC-A-70/Impact-calculations

Reply:
did we get clarity here? Can move any decisions into a new ticket if needed

Reply:
This should work for method 2a & 2b. You can see a map of method number to name in the docs linked above. Added a second table with details of func to call to use that method. But for science side those details are in the vignette. From the vignette it looks to me like this should be exported.